apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -3,7 +3,7 @@
 import { CollapseIcon, Icon } from "@/components/icon";
```

Comment:
![image](https://github.com/user-attachments/assets/6769353b-35ed-4bcf-9417-1af950ae7ca4)
The focus state is broken here.